### DISCUSSION TITLE
LSPosed that can be flashed via custom recovery

### DISCUSSION THREAD

**Discussion by PrinceSangwan** [LSPosed/LSPosed#1935] [Q&A] [answered]
Ever since google decided to ruin root experience for users, I am loosing interest in it. The thing is a lot of banking apps are making it really hard to hide root nowadays and I mainly use root for only two apps : Viper4Android and Customiuizer. The former could also be used without root through custom recovery. And the rest speaks for itself. Can you please make a version that doesn't require root and can be flashed via custom recovery? (It asked for a image so I shared my desktop. Apologies)
![2022-03-31 11_45_01-Greenshot](https://user-images.githubusercontent.com/87748567/167904345-885b2dac-ab9e-467f-b155-6bfa0c8d62b7.jpg)

**Comment by JasonKhew96** [comment, marked as answer]
**LSPosed** required **Riru** or **Zygisk**
You can't install **Riru** without **Magisk**.
You can't have **Zygisk** without **Magisk**.
